web/src/pages/profile.tsx

```diff
@@ -324,6 +325,10 @@ const ProfilePage = (props: Props): ReactElement => {
     );
   };
 
+  const displayRaffleBingoGameQuestion = (): boolean => {
+    if (!business) return false;
+    return profileData.legalStructureId === "nonprofit";
+  };
```

Comment:
[boulder] Should we add a test for this?

Reply:
Should we also add tests to assert that this _doesn't_ appear on the Oscar and Dakota profiles as well?